Wigner function functionality: Fixed an error

markjeveritt opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->
Fix for an indentation error in code.

## Description
<!--- Describe your changes in detail -->
As above.

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
@diego-plan9 @ChristopheVuillot I am very sorry but an indentation error somehow got into our code. This pull request provides a fix for it.  

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

We have downloaded the revised code to a fresh location from GitHub and testing locally.

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

diego-plan9 commented on qiskit/tools/visualization.py:
```diff
@@ -571,87 +569,84 @@ def plot_wigner_function(state, res=100):
-def plot_wigner_data(wigner_data, phis=None,
-                     thetas=None, method=None,
+
+def plot_wigner_data(wigner_data, phis=None, thetas=None, method=None,
```
In a similar note to a comment above, can you check if the parameter `theta` is needed, as it seems to be unused?

diego-plan9 commented on qiskit/tools/visualization.py:
```diff
@@ -571,87 +569,84 @@ def plot_wigner_function(state, res=100):
-                     thetas=None, method=None,
+
+def plot_wigner_data(wigner_data, phis=None, thetas=None, method=None,
                      text_out=None):
```
Can you also document `text_out` in the docstring?

diego-plan9 commented on qiskit/tools/visualization.py:
```diff
@@ -661,20 +656,19 @@ def plot_wigner_data(wigner_data, phis=None,
+            method = 'plaquette'
 
     if method == 'curve':
         plot_wigner_curve(wigner_data, xaxis=phis)
```
In the next lines:

```
     elif method == 'state':
         wigner_function(wigner_data, text_out)
```

`wigner_function()` seems not to be a valid function name: can you double-check?

markjeveritt replied:
we fixed the function name

@diego-plan9 did we get everything please?

markjeveritt commented on qiskit/tools/qcvv/tomography.py:
```diff
@@ -733,77 +731,71 @@ def build_wigner_circuits(Q_program, name, phis, thetas, qubits,
         label += str(point)
-        circuit = Q_program.create_circuit(label, [qreg], [creg])
+        circuit = q_program.create_circuit(label, [qreg], [creg])
         c_index = 0
```
It is not needed - removing now

markjeveritt commented:
@diego-plan9 

Thank you for responding to us so quickly.

Sorry about the PR being issued against a previous version of the master branch - I still am getting used to git and did not realise I had done this. Did not mean to cause extra work.

Thank you so much for the rewrite of markjeveritt:master branch - we can see it took some effort from you.

We have checked the issues mentioned on the inline comments and fixed them. In all cases tis was removing unused variables as you suggested.

We have updated and tested the Jupyter notebook. This now works agains updated libraries but note we were only able to confirm against the simulator but not the experiment as the experiments timed out.

markjeveritt commented:
@diego-plan9 Ok I am sorry but I am still confused by git - please can you let me know if we need to anything? I think this is the right place to ask/act but please let me know if I have got not wrong.

diego-plan9 commented:
Thanks for the changes, Mark! They are looking on the right track at a first glance, but I have not done a full review yet - if something else is needed, we make sure to let contributors know by commenting on the PR. In general, it is not unusual that the reviews and the "overall life time" of a pull request spawns over the course of days, depending on the work load and other factors - but we will get to it eventually, no worries!

markjeveritt commented:
@diego-plan9 Understood. I am beginning to get git now - thanks for your patience.

diego-plan9 commented:
It's looking good, thanks for the contribution, Mark! I'll merge this PR and we can continue discussion if needed on the PR for the tutorials - please note that PR might take longer to get merged, as some changes to that repository are in progress.
